<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>267</width>
    <height>293</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>267</width>
    <height>293</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>267</width>
    <height>293</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Eta_fon">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>0</y>
      <width>381</width>
      <height>331</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>fon_green_1tonii_game=test.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLineEdit" name="log">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="passw">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="login">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Логин</string>
    </property>
   </widget>
   <widget class="QLabel" name="Password">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Пароль</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_log">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>войти</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Register">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>регистрация</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pashalka">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>10</y>
      <width>1</width>
      <height>1</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLabel" name="wrong">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
